src/main/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionContext.java

```diff
@@ -484,12 +478,14 @@ private VP8FrameProjection createInLayerProjection(@NotNull VP8Frame frame,
         }
         else
         {
-            do {
+            do
+            {
                 f2 = prevFrame(f1);
```

Comment:
This will probably conflict when you rebase on master, so you might want to do that sooner rather than later.